inst/rmarkdown/templates/thesis/skeleton/skeleton.Rmd

```diff
@@ -41,10 +41,16 @@ mini-lof: false  # mini-list of figures by start of each chapter?
 ####################
 ## Bibliography   ##
 ####################
-bibliography: bib/references.bib
+bibliography: bib/references.bib #Set your bibliography file here.
+packagebibliography: bib/packages.bib #Sets the automatically generated R-Package Bibliography.
 bibliography-heading-in-pdf: References
-bib-humanities: true   #set to true if you want in-text references formatted as author-year
-doi-in-bibliography: true #set to true if you want DOI's to be shown in the bibliography
+bib-style: authoryear # See https://www.overleaf.com/learn/latex/biblatex_citation_styles for a list of the commonly used built-in citations styles of biblatex.
+bib-sorting: nyt #See https://www.overleaf.com/learn/latex/Articles/Getting_started_with_BibLaTeX for different bibliography sorting options.
+doi-in-bibliography: true #set to true if you want DOI's to be shown in the bibliography.
+bib-isbn: true #Set to true if you want the ISBN to be displayed
+bib-dashed: false #Set to true if you want dashed entries instead of repeaded author names if your bibliography contains two entries of the same author.
+bib-smallfont: false #Set to true if you want a small fontsize for your bibliography.
+citeall: true #Set this to true if you want all elements in your .bib file to appear in your bibliography (i.e. all R-packages you used).
```

Comment:
Too many options? Keep it simple by not listing these as options, and checking for them in the pandoc/latex template. If they are not there, we use sensible defaults.

Reply:
I don't necessarily agree on this one in the sense that the `{iheiddown}` package is aimed at people who do quantitative analyses and in my opinion value the customizability of the template. It's a bit like the macOS vs Windows or Linux debate; do you prefer something that works out of the box or do you prefer something that you can easily customize?